aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -2260,7 +2260,7 @@ std::tuple<Tensor&, Tensor&> linalg_eigh_out_info(
     Tensor& vectors,
     Tensor& infos,
     bool compute_eigenvectors,
-    const std::string& uplo_str) {
+    const c10::string_view& uplo_str) {
```

Comment:
const ref, weird. @swolchok do you know if your "don't feel bad about const ref" applies here?

Reply:
`c10::basic_string_view` looks [trivial for purposes of calls](https://itanium-cxx-abi.github.io/cxx-abi/abi.html#non-trivial) to me, so it should be passed by value. If anybody would like, I can check assembly to make sure.

Reply:
I'll change it to pass by value